sdk/identity/azure-identity/azure/identity/_persistent_cache.py

```diff
@@ -50,12 +53,16 @@ def __init__(self, *, allow_unencrypted_storage: bool = False, name: str = "msal
         self.name = name
```

Comment:
Should we store the is_cae information in this object?

Reply:
Right, the TokenCachePersistenceOptions would be agnostic to the cae info.